AltUseVendors/__init__.py

```diff
@@ -76,25 +76,41 @@ class ShopInfo:
 # Sound AkEvents for playing
 AKE_BUY: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Buy"
 AKE_SELL: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Sell"
-AKE_DICT = {
+AKE_VOICELINE_BY_VENDOR = {
     Game.BL2: {
         "InteractiveObj_VendingMachine_GrenadesAndAmmo": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
         "InteractiveObj_VendingMachine_HealthItems": "Ake_VOCT_Contextual.Ak_Play_VOCT_Zed_Store_Welcome",
-        #"VendingMachine_Weapons_Definition": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Bye",
-        # DLC Vendors
         "IO_Aster_VendingMachine_GrenadesAndAmmo": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Marcus_Store_Purchase",
         "IO_Aster_VendingMachine_HealthItems": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Zed_Store_Purchase",
-        #"VendingMachine_Aster_Weapons_Definition": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Marcus_Store_Bye",
-        #"VendingMachine_TorgueToken": "Ake_Iris_VO.Ak_Play_Iris_TorgueVendingMachine_Purchase",
     },
     Game.TPS: {
         "InteractiveObj_VendingMachine_GrenadesAndAmmo": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
         "InteractiveObj_VendingMachine_HealthItems": "Ake_Cork_VOCT_Contextuals.Cork_VOCT_NurseNina.Ak_Play_VOCT_Cork_NurseNina_Store_Purchase",
+        "InteractiveObj_VendingMachine_GrenadesAndAmmo_Marigold": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
+        "InteractiveObj_VendingMachine_HealthItems_Marigold": "Ake_Cork_VOCT_Contextuals.Cork_VOCT_NurseNina.Ak_Play_VOCT_Cork_NurseNina_Store_Purchase",
+        "InteractiveObj_VendingMachine_GrenadesAndAmmo_Marigold_BL1": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
+        "InteractiveObj_VendingMachine_HealthItems_Marigold_BL1": "Ake_Marigold_VOCT_Contextuals.Dlc_Marigold_VOCT_Zed.Ak_Play_Dlc_Marigold_VOCT_Zed_Vending_Welcome",
     },
     Game.AoDK: {
-        # Don't know other game AkEvent names
+        "IO_Aster_VendingMachine_GrenadesAndAmmo": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Marcus_Store_Purchase",
+        "IO_Aster_VendingMachine_HealthItems": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Zed_Store_Purchase",
     }
-}
+}[Game.GetCurrent()]
+LOADED_AKEVENTS = {}
+
+def find_and_play_akevent(caller:unrealsdk.UObject, eventName: str) -> None:
```

Comment:
`caller:unrealsdk.UObject` - caller is a bit of a useless name, it's convention for use in hooks but doesn't tell you anything here You're also not actually passing the caller, you're actually passing in a `pawn: WillowPlayerPawn` - though everywhere else in the mod uses a controller, so I changed it to that instead.

`eventName` - follow pep8, use camel case `event_name`